extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -40,3 +40,13 @@ amp-sticky-ad {
 .-amp-sticky-ad-layout > amp-ad {
   display: block;
 }
+
+.-amp-sticky-ad-close-button {
+  position: absolute;
+  height: 33px;
```

Comment:
It doesn't have to be this number. We definitely need to do some UX study to find a number here.